x/blob/types/events.go

```diff
@@ -1,23 +1,13 @@
 package types
 
-import (
-	"strconv"
+import "github.com/cosmos/gogoproto/proto"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
+var EventTypePayForBlob = proto.MessageName(&EventPayForBlob{})
 
-const (
-	EventTypePayForBlob = "payforblob"
-
-	AttributeKeySigner = "signer"
-	AttributeKeySize   = "size"
-)
-
-// NewPayForBlobEvent constructs a new payforblob sdk.Event
-func NewPayForBlobEvent(signer string, size uint64) sdk.Event {
-	return sdk.NewEvent(
-		EventTypePayForBlob,
-		sdk.NewAttribute(AttributeKeySigner, signer),
-		sdk.NewAttribute(AttributeKeySize, strconv.FormatUint(size, 10)),
-	)
+// NewPayForBlobEvent constructs a new pay for blob sdk.Event
```

Comment:
[blocking] `sdk.Event` appears incorrect here

```suggestion
// NewPayForBlobEvent returns a new EventPayForBlob
```